packages/orchestration/src/orchestration-client.test.ts

```diff
@@ -1,924 +1,926 @@
-          }
-        ],
-        tools: [addNumbersTool, multiplyNumbersTool]
-      }
-    };
-
-    const mockResponse = await parseFileToString(
-      'orchestration',
-      'orchestration-chat-completion-stream-tools-chunks.txt'
-    );
-
-    mockInference(
-      {
-        data: constructCompletionPostRequest(config, undefined, true)
-      },
-      {
-        data: mockResponse,
-        status: 200
-      },
-      {
-        url: 'inference/deployments/1234/completion'
-      }
-    );
-
-    const response = await new OrchestrationClient(config).stream();
-
-    for await (const _ of response.stream) {
-      /* empty */
-    }
-
-    const tools = response.getToolCalls();
-
-    expect(tools).toMatchInlineSnapshot(`
-     [
-       {
-         "function": {
-           "arguments": "{"a": 2, "b": 3}",
-           "name": "add",
-         },
-         "id": "call_HPgxxSmD2ctYfcJ3gp1JBc7i",
-         "index": 0,
-         "type": "function",
-       },
-       {
-         "function": {
-           "arguments": "{"a": 2, "b": 3}",
-           "name": "multiply",
-         },
-         "id": "call_PExve0Dd9hxD8hOk4Uhr1yhO",
-         "index": 1,
-         "type": "function",
-       },
-     ]
-    `);
-  });
-  describe('OrchestrationClient Stream Error Handling', () => {
-    it('should abort controller and re-throw error when network request fails', async () => {
-      const config: OrchestrationModuleConfig = {
-        llm: {
-          model_name: 'gpt-4o',
-          model_params: {}
-        },
-        templating: {
-          template: [
-            {
-              role: 'user',
-              content: 'Test prompt'
-            }
-          ]
-        }
-      };
-
-      const controller = new AbortController();
-
-      // Mock network failure
-      mockInference(
-        {
-          data: constructCompletionPostRequest(config, undefined, true)
-        },
-        {
-          status: 500,
-          data: { error: 'Internal Server Error' }
-        },
-        {
-          url: 'inference/deployments/1234/completion'
-        }
-      );
-
-      const client = new OrchestrationClient(config);
-
-      await expect(client.stream(undefined, controller)).rejects.toThrow();
-    });
-
-    it('should throw error when stream is called with already aborted controller', async () => {
-      const config: OrchestrationModuleConfig = {
-        llm: {
-          model_name: 'gpt-4o',
-          model_params: {}
-        },
-        templating: {
-          template: [
-            {
-              role: 'user',
-              content: 'Test prompt'
-            }
-          ]
-        }
-      };
-
-      const controller = new AbortController();
-
-      // Abort immediately
-      controller.abort();
-
-      const client = new OrchestrationClient(config);
-
-      await expect(client.stream(undefined, controller)).rejects.toThrow();
-    });
-  });
-});
+// import nock from 'nock';
```

Comment:
[q] I don't get why we're commenting these tests out? New test files will be creating in the new package anyway right? Later we just delete everything in this folder.